frontend/src/scenes/setup/PreflightCheck.js

```diff
@@ -0,0 +1,236 @@
+            localStorage.setItem('preflightMode', mode)
+        } else {
+            localStorage.removeItem('preflightMode')
+        }
+    }
+
+    const handlePreflightFinished = () => {
+        window.location.href = '/setup_admin'
+    }
+
+    useEffect(() => {
+        const mode = localStorage.getItem('preflightMode')
+        if (mode) handleModeChange(mode)
+    }, [])
+
+    return (
+        <>
+            <Space direction="vertical" className="space-top" style={{ width: '100%' }}>
+                <h1 className="title text-center" style={{ marginBottom: 0 }}>
+                    Welcome to PostHog!
+                </h1>
+                <div className="page-caption text-center">Understand your users. Build a better product.</div>
+            </Space>
+            <Col xs={24} style={{ marginTop: 60 }}>
+                <h2 className="subtitle text-center space-top">
+                    We're glad to have you here! We'll now guide you to setting up PostHog.
+                </h2>
+            </Col>
+            <Row style={{ display: 'flex', justifyContent: 'center' }}>
+                <div style={{ display: 'flex', alignItems: 'center', flexDirection: 'column' }}>
+                    <img src={hedgehogBlue} style={{ maxHeight: '100%' }} />
+                    <Button type="default" data-attr="support" data-source="preflight">
+                        <a href="https://posthog.com/support" target="_blank" rel="noreferrer">
+                            Support
+                        </a>
+                    </Button>
+                    <div className="breadcrumbs space-top">
+                        <span className="selected">Pre-flight check</span> &gt; Event capture &gt; Team setup
+                    </div>
+                </div>
+                <div
+                    style={{
+                        display: 'flex',
+                        justifyContent: 'center',
+                        marginLeft: 32,
+                        flexDirection: 'column',
+                        paddingTop: 32,
+                    }}
+                >
+                    <Card style={{ width: 600, width: '100%' }}>
+                        <Row style={{ marginBottom: 16 }}>
+                            <b style={{ fontSize: 16 }}>
+                                Preflight check{' '}
+                                {state.mode && (
+                                    <span>
+                                        (
+                                        <span
+                                            style={{ textDecoration: 'underline', cursor: 'pointer' }}
+                                            onClick={() => handleModeChange(null)}
+                                        >
+                                            {state.mode}
+                                        </span>
+                                        )
+                                    </span>
+                                )}
```

Comment:
![Back](https://user-images.githubusercontent.com/4550621/90757374-d789bf00-e2dd-11ea-96f0-52f611dfdb47.png)
This could be a bit nicer and more intuitive as maybe sort of a nav:
> <a href="#">Select Preflight Mode<a> > Experimentation
Where clicking "Select Preflight Mode" would take you back to that step.

Reply:
Updated.
<img width="300" src="https://user-images.githubusercontent.com/5864173/90819513-b0d78280-e2f5-11ea-917c-56041f571d7b.png">